docs/static/spec/rasa.yml

```diff
@@ -1860,3 +1881,50 @@ components:
           type: string
           description: Text of the message
           example: "Hello!"
+  examples:
+    YAMLTrainingRequest: |
+      intents:
+        - greet
+        - goodbye
+
+      responses:
+        utter_greet:
+        - text: "Hey! How are you?"
```

Comment:
Same on this section: reorder by config.yml then domain.yml then data, and add all keys even if they're empty e.g. `actions: []`

Reply:
> Same on this section: reorder by config.yml then domain.yml then data, and add all keys even if they're empty e.g. `actions: []`

@melindaloubser1 I don't think actions are used here. I don't understand the ordering, why would it `session_config` come before others? Or do you mean `pipeline` comes first, then `policies`?

Reply:
For this particular example, there are no custom actions to pass, so you are correct. However, in a bot with custom actions, or slots, or entities, these sections would need to be included.

Re. the ordering: Sorry for the shorthand, I meant the contents of config.yml, so yes, pipeline + policies, then domain content (intents, entities, actions, e2e_actions, forms, slots, responses, config<- this is a key in the domain in 3.0, which is quite confusing) then content of nlu, rules and stories. It's not required to do it this way for the example to work, but will make it much easier for readers to understand the example

Reply:
@melindaloubser1 Yeah I meant do you know custom actions are used if provided in request body like here? I don't see that in code, and I don't see them being tested in unit tests. I wouldn't be surprised if not all CLI options were supported here.

Reply:
They would have to be supported to train on stories which include custom actions I believe